lib/resty/template.lua

```diff
@@ -291,15 +291,15 @@ local function new(template, safe)
                 render = function(self, context)
                     context = context or self
                     context.blocks = context.blocks or {}
-                    context.view = template.process(view, context)
+                    context.view = function() return template.process(view, context) end
```

Comment:
Should we make it possible to override the `context`: `{* view { foo = "bar" } *}`?
```suggestion
                    context.view = function(ctx) return template.process(view, ctx, context) end
```